documents/data-sharing.md

```diff
@@ -0,0 +1,58 @@
+# Annual survey data sharing
+
+This document describes how data from the annual survey should be shared.
+
+## Objectives
+
+In priority order:
+
+* Protect the privacy of respondents.
+* Provide high quality data (unbiased data which is unlikely to be misinterpreted, c.f., raw data).
+* Provide useful and actionable data to the Rust project (Core team, Foundation, teams and working groups, contributors).
```

Comment:
I don't think it's worth singling out the core team or Foundation here -- it has as much use for the data as other Rust teams. If there is something to emphasize, it might be a prioritization of the team/working group/project group access to data over individuals (e.g., I might see us asking each team to submit a list of questions, but that the intent is that those questions are related to their work as a team).

Reply:
Hmm, I was worried that not writing Foundation explicitly might exclude it since it is not technically a team or working group. Core team I guess is duplication. To be clear this list is just spelling out who exactly is included in 'the Rust project', it's not meant to be in priority order.